aries/aries_vcx_core/src/wallet/askar/askar_utils.rs

```diff
@@ -0,0 +1,72 @@
+use aries_askar::kms::{KeyAlg, LocalKey};
+use public_key::Key;
+use serde::Deserialize;
+
+use crate::{
+    errors::error::{AriesVcxCoreErrorKind, VcxCoreResult},
+    wallet::{askar::AriesVcxCoreError, utils::random_seed},
+};
+
+pub fn local_key_to_bs58_name(local_key: &LocalKey) -> VcxCoreResult<String> {
+    let res = local_key_to_bs58_public_key(local_key)?;
+    Ok(res[0..16].to_string())
+}
+
+pub fn local_key_to_bs58_public_key(local_key: &LocalKey) -> VcxCoreResult<String> {
+    Ok(bs58::encode(local_key.to_public_bytes()?).into_string())
+}
+
+pub fn local_key_to_private_key_bytes(local_key: &LocalKey) -> VcxCoreResult<Vec<u8>> {
+    Ok(local_key.to_secret_bytes()?.to_vec())
+}
+
+pub fn local_key_to_public_key_bytes(local_key: &LocalKey) -> VcxCoreResult<Vec<u8>> {
+    Ok(local_key.to_public_bytes()?.to_vec())
+}
```

Comment:
These functions don't really need to exist - both of these are converting `SecretBytes` to `Vec<u8>` but then in practice, Vec<u8> is used as `&[u8]` slices.
And askar's `SecretBytes` is basically kind of smart pointer, it implements `Deref` for `[u8]` target. So for example in `pack_anoncrypt_recipients`  instad of :
```
let enc_key_secret = local_key_to_private_key_bytes(enc_key)?;
self.crypto_box.sealedbox_encrypt(&recipient_public_bytes, &enc_key_secret)?;
```

you can just directly get slice of secret bytes from `enc_key`
```
self.crypto_box.sealedbox_encrypt(&recipient_public_bytes, &enc_key.to_secret_bytes()?)?;
```